docs/pages/api-docs/data-grid/grid-virtualization-api.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "GridVirtualizationApi",
+  "description": "The virtualization API interface that is available in the grid.",
+  "properties": [
+    {
+      "name": "scroll",
+      "description": "Triggers the viewport to scroll to the given positions (in pixels).",
+      "type": "(params: Partial<GridScrollParams>) => void"
+    },
+    {
+      "name": "scrollToIndexes",
+      "description": "Triggers the viewport to scroll to the cell at indexes given by <code>params</code>. Returns <code>true</code> if the grid had to scroll to reach the target.",
+      "type": "(params: Optional<GridCellIndexCoordinates, 'rowIndex'>) => boolean"
```

Comment:
I notice `Optional` is a custom utility type we created:
https://github.com/mui-org/material-ui-x/blob/4f52c84e104dd598e454e5d8d0c395c5aa3b7d54/packages/grid/_modules_/grid/utils/utils.ts#L67

And we removed exporting from `@material-ui/data-grid` in #2233 . How do we document this now? 

Should we do:
```diff
- (params: Optional<GridCellIndexCoordinates, 'rowIndex'>) => boolean
+ (params: Pick<Partial<GridCellIndexCoordinates>, 'rowIndex'> & Omit<GridCellIndexCoordinates, 'rowIndex'>) => boolean
```

It's in other places in docs as well.

Reply:
We can update the type, partial is enough